Assets/Scripts/Scenes/InGame/Enemy/Trace/Feature/FeatureView.cs

```diff
@@ -13,17 +16,52 @@ public class FeatureView : MonoBehaviour
         private GameObject[] _stageInteracts;
         private GameObject[] nearStageObject = null;
         private GameObject interactTarget = null;
+        private GameObject _floor;
         AudioSource _audioSource;
         private Subject<Unit> _onDestroy = new Subject<Unit>();
         public IObservable<Unit> onDestroy { get => _onDestroy; }
+        private ReactiveProperty<GameObject> floor = new ReactiveProperty<GameObject>();
+        public IObservable<GameObject> OnFloor { get => floor; }
+        private Vector3 direction = new Vector3(0, -1, 0);
+        private StageTile _stagetile;
+        public StageTile stagetile { get { return _stagetile; } }
 
         public void Init()
         {
             _audioSource = GetComponent<AudioSource>();
             _enemy = GameObject.FindWithTag("Enemy");
             _stageInteracts = GameObject.FindGameObjectsWithTag("StageIntract");
+            StartCoroutine(FloorCheck());
         }
-        
+
+        IEnumerator FloorCheck()
+        {
+            Ray ray = new Ray(_enemy.transform.position, direction);
+            RaycastHit hit;
+            Physics.Raycast(ray.origin, ray.direction, out hit, 1.0f, 7);
+            while (true)
+            {
+                if (hit.collider.layerOverridePriority == 7)
```

Comment:
```suggestion
                if (hit.collider.layerOverridePriority == 7)
```
hit.colliderがずっとnullになってエラーが起こります。
動作確認ですが、safemodeで入る場合は問題の個所をコメントアウトすれば動作するので、その方法で正常に動作するかを確認してみて下さい

Reply:
修正しました
ご確認お願いします